src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/DropdownItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { type IconProps } from '@phosphor-icons/react';
+import React, { type ComponentType, type FC } from 'react';
+
+import { tw } from '~utils/css/index.ts';
+
+// eslint-disable-next-line max-len
+const dropdownItemClassName = tw`flex w-full items-center gap-2 rounded-s px-3 py-2 text-md text-gray-900 hover:font-medium md:hover:bg-gray-50`;
+
+interface DropdownItemProps {
+  onClick?: () => void;
+  icon: ComponentType<IconProps>;
```

Comment:
should we make this optional so we can reuse this for the teams dropdown? 
Or maybe we can refactor that down the line and have two components like
`DropdownItem` and
`DropdownIconItem` :shrug: